test/SILOptimizer/sroa.sil

```diff
@@ -23,10 +23,10 @@ sil @use_int32 : $@convention(thin) (Builtin.Int32) -> ()
 // CHECK:   [[VAR_2:%[0-9]+]] = alloc_stack $Builtin.Int32
```

Comment:
I've only moved lines up and down. I haven't changed or removed anything (beyond that).